custom_addons/crm_methodology/static/docs/sales-methodology-vs-odoo-crm.html

```
@@ -0,0 +1,719 @@
+
+      <section id="why" data-tags="s r c">
+        <div class="sec-head">
+          <span class="sec-num">02</span>
+          <h2>Why this exists</h2>
+          <span class="tags"><span class="tag s">S</span><span class="tag r">R</span><span class="tag c">C</span></span>
+        </div>
+        <p>Internal research surveyed <a href="https://claude.ai/code/artifact/9ad91be5-df4d-44fc-9327-26fa130d30dc">eight named B2B methodologies</a> — MEDDIC/MEDDPICC, Sandler, Challenger Sale, SPIN Selling, Solution Selling, CustomerCentric Selling, ValueSelling, Consultative Selling — from primary, trademark-holder sources, then checked whether stock Odoo 19 CRM already supported any of them. It found nothing: no methodology-specific fields, stages, or terminology anywhere in the vendored CRM, Sales, or Mail modules. See <a href="https://claude.ai/code/artifact/9ad91be5-df4d-44fc-9327-26fa130d30dc">The Eight B2B Sales Methodologies</a> for what each one actually claims, in its own trademark holder's words, and how it maps onto a Sales Methodology configuration.</p>
+        <p>The addon's founding commit frames the need directly: Sales Managers define named Sales Methodologies; opportunities inherit their client's methodology; reps are blocked with a clear error when a Block-enforcement Requirement is unmet. The core engine shipped first, followed by demo personas so the workflow could be shown live, and a "Reset Demo Data" action so the demo can be replayed without polluting real data.</p>
+        <pre class="mermaid">
+flowchart LR
+    A["Client's default\nmethodology"] --> B["New opportunity\ninherits it"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)